apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/SnapshotsExpirationSparkApp.java

```diff
@@ -40,42 +36,7 @@ protected void runInner(Operations ops) {
         granularity);
     long expireBeforeTimestampMs = System.currentTimeMillis() - TimeUnit.DAYS.toMillis(count);
     log.info("Expire snapshots before timestamp ms {}", expireBeforeTimestampMs);
-    ExpireSnapshots.Result result = ops.expireSnapshots(fqtn, expireBeforeTimestampMs);
-    log.info(
-        "Detected {} data files, {} manifest files, {} manifest list files that become orphaned as the result of snapshots expiration",
-        result.deletedDataFilesCount(),
-        result.deletedManifestsCount(),
-        result.deletedManifestListsCount());
-    METER
-        .counterBuilder(AppConstants.EXPIRED_FILE_COUNT)
```

Comment:
nit: Should we also remove EXPIRED_FILE_COUNT from AppConstants since they are probably not used elsewhere?

Reply:
done